cmd/zpool/zpool_main.c

```diff
@@ -4144,25 +4144,25 @@ print_zpool_script_help(char *name, char *path)
 	libzfs_free_str_array(lines, lines_cnt);
 }
 
-
 /*
- * Go though the list of all the zpool status/iostat -c scripts, run their
+ * Go though the zpool status/iostat -c scripts in path, run their
  * help option (-h), and print out the results.
  */
 static void
-print_zpool_script_list(void)
+print_zpool_dir_scripts(const char *path)
```

Comment:
You might want to call it `dirpath` instead of `path` to make it clear it the directory containing the scripts.